library/Exceptions/ValidationException.php

```diff
@@ -47,6 +47,28 @@ public static function stringify($value)
         }
     }
 
+    /**
+     * @param $value
+     * @return string
+     */
+    public static function stringifyArray($value)
+    {
+        $items = array();
+        foreach ($value as $val) {
+            if (is_object($val)) {
+                $items[] = self::stringifyObject($val);
+            } elseif (is_string($val)) {
+                $items[] = "'$val'";
+            } else {
+                $items[] = (string) $val;
```

Comment:
I've already implemented a recursive version in #387 that should handle arrays of arrays:
```
+            array(array(1, array(2, array(3))), "1, (2, (3))"),
+            array(array(1, array(2, array(3, array(4)))), "1, (2, (3, ...))"),
```
The maximum depth is a public static property.